guides/common/modules/proc_retrieving-metrics-in-the-web-ui.adoc

```diff
@@ -1,16 +1,13 @@
 [id='retrieving-metrics-in-the-web-ui_{context}']
 = Retrieving Metrics in the Web UI
 
-The Grafana web application provides a dashboard-style view, with default widgets displaying the values of metrics.
+You can view the Grafana dashboard, which provides default widgets displaying the values of metrics.
```

Comment:
this sounds off to me. Maybe "... view the Grafana dashboard to view widgets ..."?

Reply:
How about: "... view the Grafana dashboard, which displays the values of metrics as widgets." Would that work better?